### PR TITLE
[op template] embed bilingual js in HTML body

### PR DESCRIPTION
**Pull request by herohua**
@superyyrrzz @vwxyzh @hellosnow @ansyral @qinezh @chenkennt

### CODE REVIEW

**Comment by superyyrrzz** [comment]
:+1:

**Comment by chenkennt** [comment]
:shipit:

**Comment by qinezh** [comment]
:dog:

File: src/docfx.website.themes/op.html/partials/bilingualJs.tmpl.partial [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{{!Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.}}
+{{#isBilingual}}
+<div>
+  <script>OL=window.OL||{};OL.options=window.OL.options||{};
+  {{#translation_type}}
+  OL.options.translation_type='{{{translation_type}}}';
```
Comment (hellosnow) [inline src/docfx.website.themes/op.html/partials/bilingualJs.tmpl.partial]: {{ }} is enough? it shouldn't contain unescaped HTML? and {{locale}} below?

File: src/docfx.website.themes/op.html/conceptual.html.primary.js [hunk 2]
```diff
@@ -1,6 +1,13 @@
 // Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
 exports.transform = function (model) {
-  return setArrayLength(model);
+  var vm = setArrayLength(model);
+
+  if (vm.locale && vm.bilingual_type === "hover over")
```
Comment (hellosnow) [inline src/docfx.website.themes/op.html/conceptual.html.primary.js]: should we judge locale != en-US here?

Comment (herohua) [inline src/docfx.website.themes/op.html/conceptual.html.primary.js]: no need as bilingual_type would be null for en-US

File: src/docfx.website.themes/op.html/partials/bilingualJs.tmpl.partial (hunk 1 above)
Comment (herohua) [inline src/docfx.website.themes/op.html/partials/bilingualJs.tmpl.partial]: use {{{ }}} to use unescaped HTML

**Pull request merged**